wiki/quest-modding.md

```diff
@@ -109,16 +108,15 @@ Quest should work)
 
 #### Setup your Phone
 
-1. Download the [bugjaeger app from the Google Play store](https://play.google.com/store/apps/details?id=eu.sisik.hackendebug&hl=gsw&gl=US)
-2. Download the newest [BMBF APK from bmbf.dev/stable](https://bmbf.dev/stable)
-3. Follow [this written guide](https://github.com/ComputerElite/wiki/wiki/Enable-Developer-Mode-for-OQ) to Enable
+1. Download the [SideQuest app from the Google Play store](https://play.google.com/store/apps/details?id=side.quest.mobile)
+2. Follow [this written guide](https://github.com/ComputerElite/wiki/wiki/Enable-Developer-Mode-for-OQ) to Enable
```

Comment:
Could these steps be moved into the wiki so they can be translated?

Reply:
Sure. If you want to do it you can do it otherwise tell me where to move them and I'll do it

Reply:
probably best to do it in place of that step. extending the list with the steps you have on that guide